app/helpers/container_route_helper/textual_summary.rb

```diff
@@ -28,4 +28,38 @@ def textual_creation_timestamp
   def textual_resource_version
     {:label => "Resource Version", :value => @record.resource_version}
   end
+
+  def textual_ems
+    ems = @record.ext_management_system
+    return nil if ems.nil?
+    label = ui_lookup(:table => "ems_container")
+    h = {:label => label, :image => "vendor-#{ems.image_name}", :value => ems.name}
+    if role_allows(:feature => "ems_container_show")
+      h[:title] = "Show parent #{label} '#{ems.name}'"
+      h[:link] = url_for(:controller => 'ems_container', :action => 'show', :id => ems)
+    end
+    h
+  end
+
+  def textual_container_project
```

Comment:
Remove entirely this method (it has been unified somewhere else so you can just drop it).